docs/outputs.md

```diff
@@ -48,3 +48,105 @@ which will produce a ```png``` image per dump suitable for encoding into a movie
 ## Visualization software
 
 Both [ParaView](https://www.paraview.org/) and [VisIt](https://wci.llnl.gov/simulation/computer-codes/visit/) are capable of opening and visualizing Parthenon graphics dumps.  In both cases, the ```.xdmf``` files should be opened.  In ParaView, select the "XDMF Reader" when prompted.
+
+## Preparing outputs for ```yt```
+
+Parthenon HDF5 outputs can be read with the python visualization library
+[yt](https://yt-project.org/) as certain variables are named during when adding
+fields via ```StateDescriptor::AddField```. Variable names are added as a
+```std::vector<std::string>``` in the variable metadata. These labels are only
+used for output to HDF5. 4D variables are named with a list of names for each
+row while 3D variables are named with a single name.  For example, the
+following configurations are acceptable:
+
+```c++
+auto pkg = std::make_shared<StateDescriptor>("Hydro");
+
+/* ... */
+
+std::vector<std::string> cons_labels(nhydro);
+cons_labels[0]="Density";
+cons_labels[1]="MomentumDensity1";
+cons_labels[2]="MomentumDensity2";
+cons_labels[3]="MomentumDensity3";
+cons_labels[4]="TotalEnergyDensity";
+Metadata m({Metadata::Cell, Metadata::Independent, Metadata::FillGhost},
+           std::vector<int>({5}), cons_labels);
+pkg->AddField("cons", m);
+
+std::vector<std::string> density_labels(1);
+density_labels[0]="Density";
+m = Metadata({Metadata::Cell, Metadata::Derived}, std::vector<int>({1}), density_labels);
+pkg->AddField("dens", m);
+
+std::vector<std::string> velocity_labels(3);
+velocity_labels[0]="Velocity1";
+velocity_labels[1]="Velocity2";
+velocity_labels[2]="Velocity3";
+m = Metadata({Metadata::Cell, Metadata::Derived}, std::vector<int>({3}), velocity_labels);
+pkg->AddField("vel", m);
+
+std::vector<std::string> pressure_labels(1);
+pressure_labels[0]="Pressure";
+m = Metadata({Metadata::Cell, Metadata::Derived}, std::vector<int>({1}), pressure_labels);
+pkg->AddField("pres", m);
+```
```

Comment:
Maybe I'm being a bit picky, but I would recommend for the purpose of being explicit removing the magic numbers and setting them equal to a value in each of these cases.